### PR TITLE
Style the configuration cache problems report

### PR DESCRIPTION
**Pull request by eskatos**

### CODE REVIEW

**Comment by eskatos** [comment]
@bot-gradle test this

**Comment by eskatos** [comment]
@bot-gradle test this

File: subprojects/instant-execution-report/src/main/kotlin/InstantExecutionReportPage.kt [hunk 1]
```diff
@@ -263,17 +309,23 @@ object InstantExecutionReportPage : Component<InstantExecutionReportPage.Model,
...
+            if (child.tree.state === Tree.ViewState.Expanded) {
+                className("expanded")
+            }
             title("Click to ${toggleVerb(child.tree.state)}")
             onClick { treeIntent(TreeView.Intent.Toggle(child)) }
         },
         when (child.tree.state) {
-            Tree.ViewState.Collapsed -> "► "
-            Tree.ViewState.Expanded -> "▼ "
+            Tree.ViewState.Collapsed -> "⌃ "
```
Comment (bamboo) [inline subprojects/instant-execution-report/src/main/kotlin/InstantExecutionReportPage.kt:322]: ❓ Get rid of the `when` altogether?

File: subprojects/instant-execution-report/src/main/kotlin/InstantExecutionReportPage.kt [hunk 2]
```diff
@@ -118,26 +129,61 @@ object InstantExecutionReportPage : Component<InstantExecutionReportPage.Model,
...
+            className("group-selector")
+            if (tab == activeTab) {
+                className("group-selector--active")
+            } else {
+                onClick { Intent.SetTab(tab) }
+            }
+        },
+        span(
+            tab.text,
+            span(attributes { className("group-selector__count") }, "$problemsCount")
```
Comment (bamboo) [inline subprojects/instant-execution-report/src/main/kotlin/InstantExecutionReportPage.kt:186]: 💄 Put `span` arguments on separate lines.

File: subprojects/instant-execution-report/src/main/kotlin/InstantExecutionReportPage.kt [hunk 3]
```diff
@@ -86,13 +86,22 @@ object InstantExecutionReportPage : Component<InstantExecutionReportPage.Model,
         val totalProblems: Int,
         val messageTree: ProblemTreeModel,
         val taskTree: ProblemTreeModel,
-        val displayFilter: DisplayFilter = DisplayFilter.All
-    )
+        val displayFilter: DisplayFilter = DisplayFilter.All,
+        val tab: Tab = Tab.ByMessage
+    ) {
+        val messageProblemsCount = messageTree.tree.children.size
```
Comment (bamboo) [inline subprojects/instant-execution-report/src/main/kotlin/InstantExecutionReportPage.kt:92]: 💄 Introduce `ProblemTreeModel.problemCount get() = tree.children.size`

**Comment by eskatos** [comment]
@bot-gradle test this

**Comment by eskatos** [comment]
Here's a sample generated report
[5i8vjb0tma11q8wttprga08dz.zip](https://github.com/gradle/gradle/files/4806532/5i8vjb0tma11q8wttprga08dz.zip)

**Pull request merged**